src/algorithm/naive.rs

```diff
@@ -251,12 +359,12 @@ where
         // convert a percent into a fraction.
         let percent_f64 = (1.0 - ledger_view.get_damping_factors().project) * 100.0;
```

Comment:
Dampening factors need to depend on the node type now here as well. This shouldn't change the test results for now though.

Reply:
Great catch!